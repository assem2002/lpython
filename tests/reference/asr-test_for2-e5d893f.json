{
    "basename": "asr-test_for2-e5d893f",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_for2.py",
    "infile_hash": "8a05d06e94c344432594db8536e6fed71767f109b03afea0a0b80d87",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_for2-e5d893f.stderr",
    "stderr_hash": "eaaf63c17487bf1883a098fc4551c90cac26e8d0c00d8e4b27f830bd",
    "returncode": 2
}