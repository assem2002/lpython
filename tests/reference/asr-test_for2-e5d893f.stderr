semantic error: For loop increment type should be Integer.
 --> tests/errors/test_for2.py:3:5 - 4:16
  |
3 |        for i in range(0.0, 1.0, 0.4):
  |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...
...
  |
4 |            print(i)
  | ...^^^^^^^^^^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
